aie_runtime_lib/AIE/erf.h

```diff
@@ -0,0 +1,18 @@
+//===-  erf.h - get error function values for bfloat16 data type
```

Comment:
Please replace the whole file with a single comment line:
"// Unsupported erf.h for AIE1"